Added a new execution history reference sheet (#5655)
harness/developer-hub: docs/continuous-delivery/x-platform-cd-features/executions/execution-history.md

---
title: Pipeline Execution History
description: Learn more about the execution history of a pipeline. Where to find it and what information it contains.
sidebar_position: 7
---

> This topic describes how to view the execution history pipelines, how to navigate to it, and what information is contained within.

Reviewer: ```suggestion
This topic describes how to navigate to the execution history of pipelines and what information is contained within.
```

## Where Can I Find Execution History?

You can find the execution history of your pipeline or all pipelines in a project in the following places in the Harness UI.

### Executions Tab

You can view the execution history of all pipelines for a given project in the **Executions** tab in the left navigation of the Harness UI:

![](./static/leftnav-executions-tab.png)

### Pipelines Tab

You can view the execution history of all pipelines for a given project in the **Pipelines** tab also. 

In the Harness UI left navigation, select **Pipelines**. A list of pipelines created under the project appears. 

Select the more actions button on the right for the pipeline of your choice, and then select **View Executions**.

![](./static/pipelinetab-executions.png)

### Pipeline Studio

You can view the execution history of a specific pipeline using Pipeline Studio.

In the Harness UI left navigation, select **Pipelines**, and then select the pipeline for which you want to view the execution details. 

Select the **Execution History** option on the top right corner of the page to view the execution details.

![](./static/pipelinestudio-execution-history.png)

## Execution Reference

> Each execution has the following information displayed for it. Each execution also has more information if you click the arrow on the left of it and open the dropdown. 

Reviewer: ```suggestion
Each pipeline execution displays the following information.
```
